cmake/Modules/LocalAom.cmake

```diff
@@ -1,5 +1,5 @@
 set(AVIF_AOM_GIT_TAG v3.12.1)
-set(AVIF_AVM_GIT_TAG research-v9.0.0)
+set(AVIF_AVM_GIT_TAG a1aa30d1911b0815b97bb544f63bec317e903be0)
```

Comment:
research-v10.0.0-rc1 has a build system bug (https://gitlab.com/AOMediaCodec/avm/-/issues/712), so we need to check out a newer git tag with the bug fix.